log.go

```diff
@@ -61,23 +62,41 @@ func levelToName(level Level) string {
 type Logger struct {
 	Level      Level // Log messages with a level lower than this are discarded. Default level is Info
 	testing    *testing.T
-	lj         lumberjack.Logger
+	filename   string
+	log        io.Writer
 	shownError bool
 	docker     bool
 }
 
 // Create a new logger. Filename may also be one of the special names log.Stdout and log.Stderr
-func New(filename string) *Logger {
+func New(filename string, logToStdout bool) *Logger {
```

Comment:
Why would we want two ways to log to stdout? You can already specify log.Stdout as the filename here?

Reply:
This is for the case that you want to log to both stdout and to a separate file. I have found this to be very usefull during development, and it is basicly how we handle things in docker. We can remove this parameter, but I will then either change the func definition to:
```go
func New(filenames ...string) *Logger {
```
or add a separate function to force logging to stdout as well.